src/review-todo.js

```diff
@@ -0,0 +1,212 @@
+let isDate = require("date-fns/isDate");
+
+function todo(todoName = "New Todo") {
+	let title = todoName,
+		description = "",
+		urgency = 0,
+		isDone = false,
+		dueDate = todoDate(),
+		scheduleDate = todoDate(),
+		tags = tagList();
+
+	let todoObj = Object.create(todo.proto);
+	Object.assign(todoObj, {
+		title,
+		description,
+		isDone,
+		urgency,
+		dueDate,
+		scheduleDate,
+		tags,
+	});
+
+	return todoObj;
+}
+
+todo.proto = {};
+
+function project(projectName = "New Project") {
+	let proto = Object.assign(project.proto, { isChild });
+	let projectObj = Object.create(proto);
+
+	let title = projectName,
+		description = "",
+		favorite = false,
+		urgency = 0,
+		isDone = false,
+		dueDate = todoDate(),
+		scheduleDate = todoDate(),
+		tags = tagList(),
+		_todoList = createTodoList(projectObj),
+		todoList = Object.create(_todoList),
```

Comment:
if you make your `createTodoList` a constructor, you can get away from this hidden variable that you don't use anywhere else in your code. I don't know if you're familiar with the idea of "code smells" but they're things in code that look suspicious and usually point to a easier/cleaner way to write things. This line is a code smell to me.